sys/netinet/if_ether.h

```diff
@@ -35,7 +35,7 @@
 #ifndef _NETINET_IF_ETHER_H_
 #define _NETINET_IF_ETHER_H_
 
-#include <net/ethernet.h>
+//#include <net/ethernet.h>
```

Comment:
This is a FreeBSD system file. I think we should not modify it because it potentially breaks all other drivers that include it.

Maybe instead, in our source file, #define _NET_ETHERNET_H_
It's the protecting macro that is used by net/ethernet.h - so that include file won't get included.

OR maybe you just wanted to edit "our" file /usr/src/sys/compat/linuxkpi/common/include/linux/if_ether.h instead of /usr/src/sys/netinet/if_ether.h ?

Reply:
My bad - I forgot. Good catch. It was only temporary - the sys/netinet/if_ether.h is not included in the sys/dev/athn/if_athn_usb.c file, anymore.

Reply:
Adam, since I've restored the state of the file via a new commit on this branch, please either approve or provide further feedback. In the current state, the branch still fixes the RNDA-197 issue.